lib/ansiblelint/__init__.py

```diff
@@ -19,8 +19,27 @@
 # THE SOFTWARE.
 """Main ansible-lint package."""
 
-from ansiblelint.rules import AnsibleLintRule
-from ansiblelint.version import __version__
+import builtins
+from typing import Any
+
+# NOTE: Importing builtins as _builtins does not allow to redefine the
+# NOTE: type required to trick mypy into thinking that we can set the
+# NOTE: "pytest" attribute:
+_builtins: Any = builtins
```

Comment:
Sorry but adding this bit in __init__ of the tool scares the ... out of me, making me want to really avoid it. Looks like a too low level hack that can explode in our face.